### PR TITLE
Added check in IK solver, chainiksolvervel_pinv_nso, for division by zero.

### PR DESCRIPTION
**Pull request by rethink-forrest**
Added a check that the normalizing term A is not zero during the nullspace optimization step. See line 124.

### CODE REVIEW

**Comment by rethink-forrest** [comment]
It's unclear why one of the four builds on Travis CI failed the tests. The tests passed on my computer, and the tests don't actually touch the chainiksolvervel_pinv_nso solver.

I tested this on the Baxter robot over 60 user defined trajectories covering the complete workspace.

**Comment by rethink-forrest** [comment]
There was an instance of trying to access one element beyond the Twist vector which was only apparent when built with Asserts enabled.

The merge now passes the Travis CI tests.

File: orocos_kdl/src/chainiksolvervel_pinv_nso.cpp [hunk 1]
```diff
@@ -121,18 +123,20 @@ namespace KDL
             A += qd*qd * weights(i)*weights(i);
         }
 
-        // Calculate inverse Jacobian Jc^-1
-        for (i = 0; i < nj; ++i) {
-            tmp(i) = weights(i)*(q_in(i) - opt_pos(i)) / A;
-        }
+        if (A > 1e-9) {
```
Comment (smits) [inline orocos_kdl/src/chainiksolvervel_pinv_nso.cpp]: I know we have more magic numbers like this in KDL, but could you elaborate on this one? Why 1e-9 and not 1e-12 like in other places?

Comment (rethink-forrest) [inline orocos_kdl/src/chainiksolvervel_pinv_nso.cpp]: As with most magic numbers, I don't have a strong reason, but 1e-9 seemed to work well during initial testing. At that value, the difference from the goal joint configuration, must be very small.

**Pull request merged**